share/eds/ods_test.go

```diff
@@ -0,0 +1,62 @@
+package eds
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/ipld/go-car"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+// TestODSReader ensures that the reader returned from DSReader is capable of reading the CAR
+// header and ODS.
+func TestODSReader(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	// launch eds store
+	edsStore, err := newStore(t)
+	require.NoError(t, err)
+	err = edsStore.Start(ctx)
+	require.NoError(t, err)
+
+	// generate random eds data and put it into the store
+	eds, dah := randomEDS(t)
+	err = edsStore.Put(ctx, dah, eds)
+	require.NoError(t, err)
+
+	// get Car reader from store
```

Comment:
```suggestion
	// get CAR reader from store
```

Reply:
lets be more consistent maybe with our capitalization, either CAR or car. same with EDS/eds and ODS/ods. I think `Car` looks weird but idk, just an optional nit